src/core/modules/Landscape.cpp

```diff
@@ -2176,3 +2178,12 @@ float LandscapeSpherical::getOpacity(Vec3d azalt) const
 */
 	return qAlpha(pixVal)/255.0f;
 }
+
+Vec3f Landscape::horizonPolygonLineColor=Vec3f(1,0,0);
+int Landscape::horizonPolygonLineThickness=1;
+int Landscape::labelAngle=45;             // text tilt angle
+int Landscape::fontSize=12;               // Used for landscape labels (optionally indicating landscape features)
+Vec3f Landscape::labelColor=Vec3f(0,1,0); // Color for the landscape labels.
```

Comment:
Not sure where there is  conversion.

Reply:
I guess the conversion of integer literals to `float` is meant, but I'd be opposed to this request, since it reduces readability without any benefit whatsoever, even the value is preserved exactly in this conversion.